services/dy-dash/cc-rabbit-0d/src/cc-rabbit-0d/cc-rabbit-0d.py

```diff
@@ -130,59 +96,45 @@ def get_empty_cols_graphs(labelPairs=[['x', 'y']]):
     )
     return fig
 
+def get_empty_graph(xLabel='x', yLabel='y'):
+    fig = go.Figure(data=[], layout={})
+    fig = give_fig_osparc_style(fig, [xLabel], [yLabel])
+    return fig
+
+def get_empty_cols_graphs(labelPairs=[['x', 'y']]):
+    fig = tools.make_subplots(rows=1,
+                            cols=len(labelPairs),
+                            shared_xaxes=True,
+                            vertical_spacing=0.05
+    )
+    xLabels = []
+    yLabels = []
+    for labelPair in labelPairs:
```

Comment:
FYI this is notation is not pep8 . See https://gist.github.com/RichardBronosky/454964087739a449da04